mirage/utils/utils.py

```diff
@@ -157,6 +157,34 @@ def ensure_dir_exists(fullpath):
         os.makedirs(fullpath)
 
 
+def expand_environment_variable(variable_name):
+    """ Expand an environment variable and check that the directory exists
+
+    Parameters
+    ----------
+    variable_name : str
+        Environment variable name
+
+    Returns
+    -------
+    variable_directory : str
+        Path pointed to by the environment variable
+    """
+    variable_directory = os.environ.get(variable_name)
+    if variable_directory is None:
+        raise ValueError(("WARNING: {} environment variable is not set. "
```

Comment:
I suggest to remove the `WARNING:` part of the message.

Reply:
Removed.